case/src/test/kotlin/com/ritense/case/HibernateObjectMapperSupplier.kt

```diff
@@ -0,0 +1,21 @@
+package com.ritense.case
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.beans.factory.annotation.Autowired
+import java.util.function.Supplier
+
+class HibernateObjectMapperSupplier : Supplier<ObjectMapper> {
```

Comment:
Yet another solution of getting an objectMapper. Every module does it differently. One day we should fix that

Reply:
i saw it on form flow and assumed it was the way to do it.